src/main/kotlin/com/autonomousapps/DependencyAnalysisPlugin.kt

```diff
@@ -677,6 +677,20 @@ class DependencyAnalysisPlugin : Plugin<Project> {
         outputPretty.set(paths.aggregateAdvicePrettyPath)
       }
 
+    // Is there a post-processing task? If so, run it
+    afterEvaluate {
+      val postProcessingTask = if (this == rootProject) {
```

Comment:
Could we replace `afterEvaluate` if we used Gradle property or provider for postProcessingTask?

Reply:
I would have liked to do that, but I don't know if a `TaskProvider<Something>` can be held in a property.

There's also the issue that I do this wiring at configuration time and not task execution time, so I have to "wait" (via `afterEvaluate`) for the value to be set on the extension before I can query it.